### ISSUE TITLE
Clarify why no background access

### ISSUE THREAD

**Issue by tomayac** [rayankans/app-shortcuts#8]
The [current explanation](https://github.com/rayankans/app-shortcuts/blob/master/explainer.md#should-shortcuts-be-exposed-in-the-background-ie-in-serviceworkers) isn't clear to me.  

One could imagine a use case where "trending" data is provided regularly to a social networking site via a periodic background sync task. The shortcut to search for the top "trending" term could then be updated dynamically from the service worker in the background.

**Comment by aarongustafson** [comment]
Yeah, it may make sense to have this API be available from within a Worker and within the main thread (like Badging).

**Comment by rayankans** [comment]
This ties into #6. Under the assumption that shortcuts are tied to a manifest, I'm not sure how it would be possible to make it work from the background. A service worker can control multiple apps in theory, if a developer sets dynamic shortcuts from there, which app would it be routed to?

Other than that, I see no issue in exposing the API in the background as well.
